test-case/check-suspend-resume.sh

```diff
@@ -196,6 +200,17 @@ sleep_once()
     [ "$stats_success" -eq "$expected_stats_success" ] ||
         dump_and_die "/sys/power/suspend_stats/success is $stats_success, expected $expected_stats_success"
     check_suspend_fails || dump_and_die "some failure counts have changed"
+    # if OPT_VAL['p'] has process id, then check the process id is available in the system
+    if [ "${OPT_VAL['p']}" ]; then
+        dlogi "Check for the process status, pid: ${OPT_VAL['p']}"
+        sof-process-state.sh "${OPT_VAL['p']}" || {
+            #func_lib_lsof_error_dump "$snd"
```

Comment:
Why commented out? This is a failure and it's very likely very hard to reproduce. We want as much information as possible. This will be called only once.

Reply:
$snd is only available in check-suspend-resume-with-audio.sh
I put a FIXME comment

Reply:
So if this can never work then why have this commented out line at all?